apps/src/lib/tools/jsdebugger/redux.js

```diff
@@ -15,14 +16,22 @@ const CLEAR_LOG = 'jsdebugger/CLEAR_LOG';
 const OPEN = 'jsdebugger/OPEN';
 const CLOSE = 'jsdebugger/CLOSE';
 
+function flagOnState() {
+  if (experiments.isEnabled('react-inspector')) {
+    return [];
+  } else {
+    return '';
+  }
+}
+
 // State model
 const JSDebuggerState = Immutable.Record({
   jsInterpreter: null,
   runApp: null,
   observer: null,
   watchIntervalId: null,
   commandHistory: null,
-  logOutput: '',
+  logOutput: flagOnState(),
```

Comment:
Another good place for some simplification with a ternary.

Reply:
Awesome! I'll refactor it with ternary!